tests/cxx/using_aliased_symbol_unused.cc

```diff
@@ -0,0 +1,28 @@
+//===--- using_aliased_symbol_unused.cc - test input file for iwyu --------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// Tests that when we include a file that contains a using decl, that we are not
+// forced to include that file if the decl is not used.
+
+#include "using_aliased_symbol-alias.h"
+
+void use_symbol() {  
```

Comment:
`use_symbol` is technically not correct, better is something like `not_use_symbol`.